src/invidious/helpers/sig_helper.cr

```diff
@@ -135,7 +136,15 @@ class Invidious::SigHelper
         player_version = IO::ByteFormat::NetworkEndian.decode(UInt32, bytes[1..4])
         has_player ? player_version : nil
       end
+    end
+
+    # Return when the player was last updated
+    def get_player_timestamp : UInt64?
+      request = Request.new(Opcode::GET_SIGNATURE_TIMESTAMP, nil)
```

Comment:
this should probably be `Opcode::PLAYER_UPDATE_TIMESTAMP`, right?

Reply:
You're right, thanks!